### ISSUE TITLE
Is there a way to cache DOI lookups in the test suite

### ISSUE THREAD

**Issue by Kypore1** [pr-omethe-us/PyKED#94]
As is it takes a long time to validate all the files because it has to download them all each time. Could we look into adding something to cache the files?

**Comment by kyleniemeyer** [comment]
We did start to look at why the test suite is so slow in #86

**Comment by bryanwweber** [comment]
Indeed, and @Kypore1 was running the code from before the commits that fixed #86. Let's see if the speed improves after updating to master. I still think this might be a good idea though... we can download JSON files of the message response or something in a directory that can be "cleaned" with a separate command

**Comment by kyleniemeyer** [comment]
Sure, maybe we can set up a list of all the queries for the tests and do it all in one?

**Comment by kyleniemeyer** [comment]
#133 has replaced the live API tests with mocks, and moved those live API tests to a separate workflow that runs on a regular schedule.
